### DISCUSSION TITLE
Output VTU + ASCII

### DISCUSSION THREAD

**Discussion by hectormartineznavarro** [rsachetto/MonoAlg3D_C#38] [Q&A] [answered]
Hi!

My name is Hector Martinez-Navarro, I am a new postdoc at Blanca's group (did my PhD with her though). I am enjoying very much learning how to use MonoAlg3D, feels very easy and intuitive!

I was wondering how I can set the output options to print results both in VTU format and plain text. Probably it is very easy, but I can't make it happen from the config file. Is there a way to choose several output functions?

Nice meeting you! :)

`[save_result]  
print_rate=20
output_dir=./outputs/dti003_ctrl
init_function=init_save_as_vtk_or_vtu
end_function=end_save_as_vtk_or_vtu
main_function=save_as_vtu
save_pvd=true
binary=false
file_prefix=V`

**Comment by rsachetto** [comment, marked as answer]
Hi Hector, 

Glad you are liking the software.
Right now, it is not possible to save in vtu and ascii at the same time.  I can implement it as an option for the save_as_vtk_or_vtu function. 
For now, as a workaround, you can use the converter tool to convert between formats. For instance, if you  saved the results as vtu you can use the following command to convert them to txt:

``
./bin/MonoAlg3D_converter -i ./outputs/dti003_ctrl
``

The converted files will be saved at outputs/dti003_ctrl/converted_files/
As soon as I have implemented the txt functionality I get back to you.

Hope this helps, 
Rafael Sachetto

**Reply by hectormartineznavarro** [reply]
Thank you very much, Rafael,

that was really helpful!

Best wishes,
Hector
